src/main/java/org/folio/circulation/domain/MultipleRecords.java

```diff
@@ -113,4 +113,12 @@ public JsonObject asJson(
   public Integer getTotalRecords() {
     return totalRecords;
   }
+
+  public <R> Result<MultipleRecords<R>> map(Function<T, Result<R>> mapper) {
```

Comment:
```suggestion
  public <R> Result<MultipleRecords<R>> flatMapRecords(Function<T, Result<R>> mapper) {
```
*Rationale*
* `Records` -> This maps all of the records in the object. 
* `flatMap` -> The mapper produces a Result<R>, so a map would be MultipleRecords<Result<R>>,`flat` indicates the reduction of <Result<R>> to <R>.

Does that make sense?

Reply:
Makes sense to me, thanks for the suggestion.

Reply:
Done